test/sys/test_ptrace.rs

```diff
@@ -1,6 +1,7 @@
 use nix::Error;
 use nix::errno::*;
 use nix::unistd::*;
+use nix::sys::ptrace;
 use nix::sys::ptrace::*;
```

Comment:
This change doesn't seem so useful if you still need to glob import a ton of things. Can we get rid of those here or does that make things really cluttered?